src/components/toast/toast.stories.tsx

```diff
@@ -354,6 +354,111 @@ export const AlignedRight = () => {
   );
 };
 
+export const AlignedYTop = () => {
```

Comment:
Suggestion: I think it would be worth having some demos with the different permutations of `align` and `alignY` set